app/views/users/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <ul>
+    <% @user.errors.each do |e| %>
+      <li>Your <%= e %> is invalid. Please try something else</li>
+    <% end %>
+  </ul>
+<% end %>
+<%= form_for @user, url: user_path(@user) do |f| %>
+  Username:  <%= f.text_field :username, placeholder: 'Desired username' %><br><br>
+  E-mail: <%= f.email_field :email, placeholder: 'email@company.com' %><br><br>
+  Password: <%= f.password_field :password %><br><br>
+  <%= f.submit %>
+<% end %>
```

Comment:
- [ ] You are missing the commented out `HTML form` and `#form_tag forms` in the edit view. 
Screenshot from Odin 👇 
![Screen Shot 2020-11-04 at 1 13 17 AM](https://user-images.githubusercontent.com/48804258/98054320-ec84e180-1e3a-11eb-90d8-ed8c712a2cd3.png)